docs/UserGuide.md

```diff
@@ -882,28 +936,29 @@ If all else fails, you can try [uploading the image via GUI](#adding-by-graphica
-| [**Find commissions**](#find-a-commission-findcom)                     | `findcom [k/KEYWORD]... -all [t/TAG]... -any [t/TAG]...`                                                                                                                                                                                                                                                                              |
-| [**Help**](#viewing-help-help)                                         | `help`                                                                                                                                                                                                                                                                                                                                |
-| [**Clear everything**](#clearing-all-the-customers-clear)              | `clear`                                                                                                                                                                                                                                                                                                                               |
-| [**Exit**](#exiting-the-program-exit)                                  | `exit`                                                                                                                                                                                                                                                                                                                                |
+| Action                                                                                          | Format, Examples                                                                                                                                                                                                                                                                                                                      |
+|-------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [**Open customer/Switch to Customer Tab**](#viewing-a-customer-opencus)                         | `opencus INDEX`<br> e.g., <code>opencus <em>2</em></code>, <code>opencus</code>                                                                                                                                                                                                                                                       |                                                                                              |
```

Comment:
woops sorry missed this out:
#216 : index should be optional? `opencus INDEX` should be `opencus [INDEX]`?